e2e/pages/events/manageDocumentsLAEvent.page.js

```diff
@@ -109,11 +109,40 @@ module.exports = {
     I.attachFile(this.fields.supportingDocuments(elementIndex).document, document);
   },
 
+  async selectFurtherEvidenceType(type) {
+    const elementIndex = await this.getActiveElementIndex();
+    switch (type) {
+      case 'Expert reports':
+        I.checkOption(this.fields.supportingDocuments(elementIndex).type.expert);
+        break;
+      case 'Other reports':
+        I.checkOption(this.fields.supportingDocuments(elementIndex).type.other);
+        break;
+      default:
+        throw new Error(`Unsupported further evidence type ${type}`);
+    }
+  },
+
   async selectConfidential() {
     const elementIndex = await this.getActiveElementIndex();
     I.click(this.fields.supportingDocuments(elementIndex).confidential);
   },
 
+  async uploadFurtherEvidence(supportingEvidenceDocument) {
```

Comment:
same as before, one function can reference another, or additional default param can be added